tests/integration/pom.xml

```diff
@@ -36,6 +36,7 @@
             <artifactId>mule-core</artifactId>
             <version>${project.version}</version>
             <scope>provided</scope>
+            <exclusions><exclusion><groupId>org.zaraza</groupId><artifactId>zaraza</artifactId></exclusion></exclusions>
```

Comment:
Removed exclusion, was just for testing and I didn't realise that I committed this.